vector/src/main/java/im/vector/app/features/location/LocationSharingService.kt

```diff
@@ -143,6 +142,26 @@ class LocationSharingService : VectorService(), LocationTracker.Callback {
 
     override fun onLocationUpdate(locationData: LocationData) {
         Timber.i("### LocationSharingService.onLocationUpdate. Uncertainty: ${locationData.uncertainty}")
+
+        // Emit location update to all rooms in which live location sharing is active
+        roomArgsList.forEach { roomArg ->
+            sendLiveLocation(roomArg.roomId, locationData)
```

Comment:
is `sendLiveLocation` a blocking network call? it might need a io dispatchers~ anddd it might be safer to copy the `roomArgsList` to avoid also blocking the synchronised calls 

```kotlin
val inputs = roomArgsList.toList()

scope.launch(io) {
  inputs.forEach { sendLiveLocation() }
}
```